pyrightconfig.stricter.json

```diff
@@ -71,7 +71,8 @@
         "stubs/stripe",
         "stubs/ttkthemes",
         "stubs/urllib3",
-        "stubs/vobject"
+        "stubs/vobject",
+        "stubs/xmltodict"
```

Comment:
Is this necessary? As far as I can see, you fully typed the stubs, and if I remember correctly, incompletely typed stubs is what the stricter config will warn about.

Reply:
No idea, this was added by the script automatically. Should I remove it?

Reply:
Oh, it’s an exclusion list. Removed.